examples/kube-stack-private/kube.tf

```diff
@@ -38,76 +38,20 @@ module "kube-cluster" {
   ]
 }
 
-# security group for kube controller
 module "kube-controller-sg" {
-  source      = "../../modules/security-group-base"
+  source      = "../../modules/kube-controller-sg"
   name        = "${var.name}-kube-controller"
```

Comment:
In this case, you want to split the `${var.name}` into `${var.name_prefix}` and `${var.name_suffix}`, then give `name_suffix` a default of `kube-controller`, and set no default on `name_prefix`. That then requires the operator sets `name_prefix`, and applies a default naming scheme, but allows them to override the naming scheme if they choose.

You will see this pattern in the other modules.